plugin_manager.py

```diff
@@ -13,40 +13,69 @@ class PluginManager:
     def __init__(self):
 
         default_config = {
-            # Plugin data, priority in range 0-9 inclusive, persistent is boolean
-            'PLUGINS': {}
+            # Plugin and filter data, priority in range 0-9 inclusive, persistent only for plugins
+            'PLUGINS': {},
+            'FILTERS': {}
         }
 
         config_loader.register_config('plugin', PLUGIN_FILE_NAME, default_config)
         self.plugins = config_loader.load_config('plugin')
         self.loaded_plugins = []
-        self.blocked = {}  # Dictionary to prevent multiple additions of same plugin to queue
+        self.loaded_filters = []
+        self.blocked = {}  # Dictionary to prevent multiple additions of same plugin/filter to queue
         self.config = config_loader.load_config('twilight')
 
+    def get_files(self, location):
+        """Helper function to retrieve plugin and filter scripts"""
```

Comment:
Please describe args and return values. 
See PEP 257 / twilight.py for reference

Fix docstrings similarly elsewhere as well please.